html/data-methods.ejs

```diff
@@ -4,216 +4,168 @@
-            Argentina, Brazil and Paraguay; palm oil in Indonesia and Colombia;
-            and coffee in Colombia. Additional countries and commodities will
-            be added in subsequent years based on capacity and funding
-            opportunities.
-          </p>
-          <p class="paragraph">
-            From 2017, Trase will focus first on expanding to include all South
-            American soy, and initiating new programmes of work on beef in
-            Argentina, Brazil and Paraguay; palm oil in Indonesia and Colombia;
-            and coffee in Colombia. Additional countries and commodities will
-            be added in subsequent years based on capacity and funding
-            opportunities.
-          </p>
-          <p class="paragraph">
-            While the modelling framework and decision logic that underpin
-            SEI-PCS are transferable between different commodities and countries,
-            each version of SEI-PCS is highly tailored to the specific context,
-            requiring detailed knowledge of national production, logistics,
-            taxation and other commodity-country-specific data. This means that
-            with the current limited capacity of Trase, expanding to a new
-            commodity and country requires approximately six months. To learn
-            about plans for SEI-PCS and Trase expansion to other countries and
-            commodities please sign up for our newsletter
-          </p>
-        </div>
+        <h2 id="commodities-countries" class="title -anchor -medium -pink">What commodities and countries does Trase
+          cover?</h2>
+        <p class="paragraph">
+          Over the next five years Trase aims to cover over 70% of the total traded volume of major forest-risk
+          commodities that originate in the tropics. These include soy, beef, palm oil, timber, pulp and paper, coffee,
+          cocoa and aquaculture products.
+        </p>
+        <p class="paragraph">
+          From 2017, Trase will focus first on expanding to include all South American soy, and initiating new
+          programmes of work on beef in Argentina, Brazil and Paraguay; palm oil in Indonesia and Colombia; and coffee
+          in Colombia. Additional countries and commodities will be added in subsequent years based on capacity and
+          funding opportunities.
+        </p>
+        <p class="paragraph">
+          While the modelling framework and decision logic that underpin SEI-PCS are transferable between different
+          commodities and countries, each version of SEI-PCS is highly tailored to the specific context, requiring
+          detailed knowledge of national production, logistics, taxation and other commodity-country-specific data. To
+          learn about plans for SEI-PCS and Trase expansion to other countries and commodities please <a
+                  href="/#newsletter" target="_blank">sign up</a> for our newsletter.
```

Comment:
This leads to nothing as the inactive newsletter section of the homepage has been removed.